src/app/shell/personal-cabinet/provider/create-workshop/create-about-form/working-hours-form-wrapper/working-hours-form/working-hours-form.component.html

```diff
@@ -21,50 +21,61 @@
             <p class="text">{{ 'FROM' | translate }}</p>
             <input
               class="input step-input start-time-input"
-              type="text"
               matInput
-              [ngxMatTimepicker]="timepickerStart"
-              [format]="24"
-              readonly
-              required
+              (input)="validateTimeInput($event)"
```

Comment:
Is it possible to subscribe to formControl value change instead of input/blur? I would like to avoid using native events in reactive forms